toucan_connectors/postgres/postgresql_connector.py

```diff
@@ -124,7 +117,7 @@ def _retrieve_data(self, data_source):
 
     @staticmethod
     def _get_details(index: int, status: Optional[bool]):
-        checks = ['Hostname resolved', 'Port opened', 'Host connection', 'Authenticated']
+        checks = ['Host resolved', 'Port opened', 'Host connection', 'Authenticated']
```

Comment:
is not this redundant ?
'Host resolved', and 'Host connection'

Reply:
I had the same thought, but it's actually not! The first verify that the name or IP is valid (we do this by using `self.check_hostname`), and the second one is actually the connection using `psycopg`. It's a good idea to rename it though!

Reply:
b3579b8 clearer? :)

Reply:
Yes, Thanks for the clarification ! 👍🏾